Dockerfile/electrs/Dockerfile

```diff
@@ -35,9 +35,9 @@ RUN apt-get update -y \
 
 COPY --from=builder /usr/local/cargo/bin/electrs /usr/local/bin
 
-ENV BITCOIN_DATA=/var/lib/bitcoin
+ENV BITCOIN_DATA=/var/lib/bitcoind
 ENV ELECTRS_DATA=/var/lib/electrs
-ENV ELECTRS_CONF=/etc/electrs/electrs.toml
+ENV ELECTRS_CONF=/etc/electrs/config.toml
```

Comment:
Once we are using default config there is no need in ENV variable, which is not used anymore.

Reply:
the `ELECTRS_CONF` env variable is still used here: https://github.com/zoedberg/docker/blob/tesnet_compose/Dockerfile/electrs/Dockerfile#L43

Reply:
It could be error-prone to specify the same path in one place with environment variable and in the other place with string literal. I think we need to stick to some one way of doing it...

Reply:
I don't see `/etc/electrs/config.toml` as string literal anywhere except in the `ELECTRS_CONF` variable assignment